File: 0075-sort-colors/0075-sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        a = b = c = 0

        for x in nums:
            if x == 0:
                a += 1
            elif x == 1:
                b += 1
            else:
                c += 1

        nums[:] = [0] * a + [1] * b + [2] * c
                


        

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int a = 0, b = 0, c = 0;
        for (int x : nums) {
            if (x == 0)
                a++;
            else if (x == 1)
                b++;
            else
                c++;
        }

        int i = 0;

        while (a--)
            nums[i++] = 0;

        while (b--)
            nums[i++] = 1;

        while (c--)
            nums[i++] = 2;
    }
};